R/process_cube_functions.R

```diff
@@ -164,6 +183,12 @@ process_cube <- function(cube_name,
 
     }
 
+    # if the user has chosen 'none' as a grid type...
+  } else if (grid_type == "none") {
+
+      # create dummy column full of zeros
+      occurrence_data$cellCode <- 0
```

Comment:
I thought the issue was that there are no grid codes? In any case, I got rid of this code so you can attach your own cell codes column..

I fixed the issue with column name order. The problem was actually a coding error that mixed the column names into the wrong order rather than an assumption of any particular order.

Extra columns are already allowed, there should not be an issue there..?